lib/manager/terraform/lockfile/index.spec.ts

```diff
@@ -0,0 +1,328 @@
+import { join } from 'upath';
+import * as httpMock from '../../../../test/http-mock';
+import { getName, loadFixture } from '../../../../test/util';
+import { getPkgReleases } from '../../../datasource';
+import { defaultRegistryUrls } from '../../../datasource/terraform-provider';
+import * as _fs from '../../../util/fs';
+import type { UpdateArtifactsConfig } from '../../types';
+import hash from './hash';
+import { updateArtifacts } from './index';
+
+// auto-mock fs
+jest.mock('../../../util/fs');
+jest.mock('./hash');
+jest.mock('../../../datasource');
+
+jest.setTimeout(15000);
```

Comment:
Are the tests that slow?

Reply:
no, this a left over from when I have used bigger files and haven't mocked the FS operations. removing